scripts/create_cu_vm.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python2
+"""
+Create CU VMs on a given provider.
+Where possible, defaults will come from cfme_data
+
+Usage:
+1.scripts/create_cu_vm.py --provider vsphere55 --vm_name cu-vm --template fedora-22-server-template
+2.scripts/create_cu_vm.py --provider vsphere55 --template fedora-22-server-template
+
+If vm_name is not passed,the script creates these 2 CU VMs: cu-9-5 and cu-24x7
+"""
+import argparse
+import sys
+
+from cfme.exceptions import CUCommandException
+from utils.conf import cfme_data, credentials
+from utils.log import logger
+from utils.providers import get_mgmt
+from utils.ssh import SSHClient
+from utils.virtual_machines import deploy_template, _vm_cleanup
+
+
+def command_run(ssh_client, command, message):
+    exit_status, output = ssh_client.run_command(command)
```

Comment:
We can ignore the landscape error here, still trying to decide the best way to deal with it.